sunpy/map/tests/test_mapbase.py

```diff
@@ -480,11 +490,50 @@ def test_rotate_invalid_order(generic_map):
 
 
 @skip_wcsaxes
+def test_as_mpl_axes_aia171(aia171_test_map):
+    import wcsaxes  # import here because of skip
+    ax = plt.subplot(projection=aia171_test_map)
+    assert isinstance(ax, wcsaxes.WCSAxes)
+    # This test doesn't work, it seems that WCSAxes copies or changes the WCS
+    # object.
```

Comment:
what do you mean it doesn't work? that it fails? should then use the xfail thingy from pytest?